### ISSUE TITLE
Skaffold commands fail when Minikube cluster (v1.12.0-beta.0) is the current context.

### ISSUE THREAD

**Issue by fsladkey** [GoogleContainerTools/skaffold#4418]
# Skaffold commands fail when Minikube cluster (v1.12.0-beta.0) is the current context.

Ran into this issue today after updating to the latest version of the Cloud SDK which has minikube version v1.12.0-beta.0.

# Expected behavior

Running `skaffold dev` runs or shows a meaningful error

### Actual behavior

`skaffold dev` exits immediatly with the error
```
 skaffold dev
creating runner: creating builder: getting docker client: unable to parse minikube docker-env keyvalue: [# To point your shell to minikube's docker-daemon, run:], line: # To point your shell to minikube's docker-daemon, run:, output: DOCKER_TLS_VERIFY=1
DOCKER_HOST=tcp://127.0.0.1:32781
DOCKER_CERT_PATH=/Users/{home}/.minikube/certs
MINIKUBE_ACTIVE_DOCKERD=minikube

# To point your shell to minikube's docker-daemon, run:
# eval $(minikube -p minikube docker-env)
```

running `eval $(minikube -p minikube docker-env)` does not change the behavior.

### Information

- Skaffold version: version 1.12.0
- Operating system: Mac OSX
- Contents of skaffold.yaml:

```yaml
apiVersion: skaffold/v1beta15
kind: Config
build:
  tagPolicy:
    sha256: {}
  artifacts:
    - image: java-hello-world
      jib: {}
deploy:
  kubectl:
    manifests:
    - ./kubernetes-manifests/**.yaml

```

### Steps to reproduce the behavior

1. Install minikube v1.12.0-beta.0 (version bundled with latest gcloud)
2. Clone [a Cloud Code sample](https://github.com/GoogleCloudPlatform/cloud-code-samples/tree/master/java/java-hello-world) (any should work)
2. Set current context to minikube
3. `skaffold dev`

**Comment by balopat** [comment]
I believe the issue is on the minikube side: https://github.com/kubernetes/minikube/issues/8615

**Comment by dgageot** [comment]
I'll add a fix to skaffold to workaround this issue

**Comment by tejal29** [comment]
Should be fixed by #4421
